.github/workflows/tests.yml

```
@@ -0,0 +1,150 @@
+          files: ./backend/coverage.out,./sdk/sdk-typescript/coverage/lcov.info
+          flags: full-tests
+          name: full-tests-coverage
+
+      - name: Upload security scan results
+        uses: github/codeql-action/upload-sarif@v2
+        if: always()
+        with:
+          sarif_file: backend/gosec-report.json
+
+      - name: Upload test artifacts
+        uses: actions/upload-artifact@v3
```

actionlint (1.7.12)

[error] the runner of "actions/upload-artifact@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)